anagram.cpp

```diff
@@ -1,13 +1,61 @@
 #include "anagram.h"
+#include<algorithm>
+using namespace std;
 
-bool Anagram::WordPairIsAnagram(const std::string& word1, const std::string& word2) {
-    //Fill the correct implementation here
+
+string arrange(string& str)
+{
```

Comment:
Good approach to isolate the pre-processing of strings into an 'arrange' function 👍 

Suggestion: you can use iterators to remove spaces too.